src/ImageSharp/ImageSharp.csproj

```
@@ -20,6 +21,11 @@
         <TargetFrameworks>net5.0;netcoreapp3.1;netcoreapp2.1;netstandard2.1;netstandard2.0;netstandard1.3;net472</TargetFrameworks>
       </PropertyGroup>
     </When>
+    <When Condition="$(Configuration.EndsWith('InnerLoop')) == true">
+      <PropertyGroup>
+        <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
```

How about this?
https://github.com/adamralph/minver#can-i-disable-minver

```suggestion
        <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
        <MinVerSkip>true</MinVerSkip>
```

---

Didn't know that was possible! I'll leave the changes in the submodule for now though as I don't really see a use for MinVer outside of CI.